apps/ledger-live-desktop/src/renderer/screens/nft/Gallery/TokensList/Item.jsx

```diff
@@ -94,14 +102,14 @@ const NftCard = ({ id, mode, account, withContextMenu = false, onHideCollection
         className={show || process.env.ALWAYS_SHOW_SKELETONS ? "disabled" : ""}
         horizontal={!isGrid}
         alignItems={!isGrid ? "center" : undefined}
-        onClick={onItemClick}
+        onClick={overloadOnItemClick ? () => overloadOnItemClick(metadata) : onItemClick}
       >
         <Skeleton width={40} minHeight={40} full={isGrid} show={show}>
           <Media
             metadata={metadata}
             tokenId={nft.tokenId}
-            size={40}
-            full={isGrid}
+            size={overloadOnItemClick ? 180 : 40}
```

Comment:
It seems weird to link the onClick overloading to style properties, separate it instead